src/docfx/Errors.cs

```diff
@@ -86,8 +86,15 @@ public static Error GitHubUserNotFound(string login)
         ///   - the api call reach github limit
         ///   - using invalid access token(more detailed info in ex.Message)
         /// </summary>
-        public static Error GitHubApiFailed(string api, Exception ex)
-            => new Error(ErrorLevel.Warning, "github-api-failed", $"Failed calling GitHub API '{api}': {ex.Message}");
+        public static Error GitHubApiFailed(string api, Exception ex, int? retryAfterSeconds = null)
```

Comment:
Don't use error messages to log diagnostics, if you want to log just `Log.Write()`